RHEL 8.8 backported the same change as RHEL 9.2, so the check that is needed for RHEL 9.2 is also needed for RHEL 8.8.
See: https://github.com/amzn/amzn-drivers/issues/271

--- a/ena/kcompat.h
+++ b/ena/kcompat.h
@@ -1093,7 +1093,7 @@ static inline void ena_netif_napi_add(struct net_device *dev,
                                      int (*poll)(struct napi_struct *, int))
 {
 #if (LINUX_VERSION_CODE < KERNEL_VERSION(6, 1, 0)) && \
-       !(RHEL_RELEASE_CODE && (RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(9, 2)))
+       !(RHEL_RELEASE_CODE && (RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(9, 2) || RHEL_RELEASE_CODE == RHEL_RELEASE_VERSION(8, 8)))
 #ifndef NAPI_POLL_WEIGHT
 #define NAPI_POLL_WEIGHT 64
 #endif
